Source/WebCore/accessibility/AccessibilityMenuListPopup.h

```diff
@@ -63,7 +63,11 @@ class AccessibilityMenuListPopup final : public AccessibilityMockObject {
 
 } // namespace WebCore
 
-SPECIALIZE_TYPE_TRAITS_BEGIN(WebCore::AccessibilityMenuListPopup) \
-    static bool isType(const WebCore::AccessibilityObject& object) { return object.isMenuListPopup(); } \
-    static bool isType(const WebCore::AXCoreObject& object) { return object.isAccessibilityObject() && downcast<WebCore::AccessibilityObject>(object).isMenuListPopup(); } \
+SPECIALIZE_TYPE_TRAITS_BEGIN(WebCore::AccessibilityMenuListPopup)
+    static bool isType(const WebCore::AccessibilityObject& object) { return object.isMenuListPopup(); }
+    static bool isType(const WebCore::AXCoreObject& object)
+    {
+        auto* accessibilityObject = dynamicDowncast<WebCore::AccessibilityObject>(object);
+        return accessibilityObject && accessibilityObject->isMenuListPopup();
+    }
```

Comment:
Same comment about these lines being indented.

Reply:
We should match the rest of the code base for these traits. (I suspect they are indented).

Reply:
They are, as far as I can tell. At least a very large amount of them.